app/Models/Context.php

```diff
@@ -94,6 +95,13 @@ public static function currentGet($array = false) {
 	}
 
 	/**
+	 * Return true iif the current requests target a feed and not a category or all articles.
```

Comment:
So I would prefer "if and only if", I'm not a fan of abbreviations in comments ;)